src/app.js

```diff
@@ -0,0 +1,29 @@
+import './assets/scss/app.scss';
+import $ from 'cash-dom';
+
+
+export class App {
+  initializeApp() {
+    let self = this;
+
+    $('.load-username').on('click', function (e) {
```

Comment:
1. Consider storing `.load-username` in some variable, as elements' selectors are often used multiple times (as we have here, in line 24). 

2. Anonymous function could be replaced with arrow function. Also, callback's parameter is not used, so there is no need to declare it.